app/src/main/res/layout/fragment_tab_tray.xml

```diff
@@ -32,67 +31,73 @@
-        app:layout_constraintBottom_toTopOf="@id/bottomBarShadow">
+        app:layout_constraintBottom_toTopOf="@id/bottomBarShadow"
+        app:layout_constraintTop_toBottomOf="@+id/tabsTray">
+
         <include
-            layout="@layout/save_to_collection_button"
-            android:id="@+id/saveToCollectionButton"/>
+            android:id="@+id/saveToCollectionButton"
+            layout="@layout/save_to_collection_button" />
     </LinearLayout>
 
     <View
         android:id="@+id/bottomBarShadow"
         android:layout_width="0dp"
         android:layout_height="1dp"
         android:background="@color/bottom_bar_shadow"
+        app:layout_constraintBottom_toTopOf="@id/tab_tray_controls"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintBottom_toTopOf="@id/tab_tray_controls" />
+        app:layout_constraintStart_toStartOf="parent" />
 
-    <LinearLayout
+    <View
         android:id="@+id/tab_tray_controls"
-        android:layout_height="48dp"
         android:layout_width="0dp"
-        android:orientation="horizontal"
-        android:gravity="center"
+        android:layout_height="48dp"
         android:background="?bottomBarBackground"
-        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintBottom_toBottomOf="parent">
-        <ImageButton
-            android:id="@+id/private_browsing_button"
-            android:layout_width="@dimen/glyph_button_height"
-            android:layout_height="@dimen/glyph_button_height"
-            android:background="?android:attr/selectableItemBackgroundBorderless"
-            android:contentDescription="@string/content_description_private_browsing_button"
-            app:srcCompat="@drawable/private_browsing_button"
-            android:layout_weight=".3333" />
-        <ImageButton
-            android:id="@+id/tab_tray_go_home"
-            android:layout_width="@dimen/glyph_button_height"
-            android:layout_height="@dimen/glyph_button_height"
-            android:background="?android:attr/selectableItemBackgroundBorderless"
-            android:contentDescription="@string/tab_tray_menu_home"
-            app:srcCompat="@drawable/ic_home"
-            android:layout_weight=".3333" />
-        <ImageButton
-            android:id="@+id/tab_tray_open_new_tab"
-            android:layout_width="@dimen/glyph_button_height"
-            android:layout_height="@dimen/glyph_button_height"
-            android:background="?android:attr/selectableItemBackgroundBorderless"
-            android:contentDescription="@string/tab_tray_menu_open_new_tab"
-            app:srcCompat="@drawable/ic_new"
-            android:layout_weight=".3333" />
-    </LinearLayout>
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <ImageButton
```

Comment:
should we also constrain the top of all of these to the top of tab_tray_controls to make sure they're vertically centered in that view?